packages/editor/src/components/entities-saved-states/index.js

```diff
@@ -49,18 +114,22 @@ function EntityTypeList( { list, unselectedEntities, setUnselectedEntities } ) {
 						onChange={ ( value ) =>
 							setUnselectedEntities( record, value )
 						}
+						closePanel={ closePanel }
 					/>
 				);
 			} ) }
-		</div>
+		</PanelBody>
 	);
 }
 
-export default function EntitiesSavedStates( { isOpen, onRequestClose } ) {
-	const dirtyEntityRecords = useSelect(
-		( select ) => select( 'core' ).__experimentalGetDirtyEntityRecords(),
-		[]
-	);
+export default function EntitiesSavedStates( { isOpen, closePanel } ) {
```

Comment:
```suggestion
export default function EntitiesSavedStates( { isOpen, close } ) {
```

Would be cleaner.

Reply:
We have `close` in the upper scope as an icon import, we could import it under a different name but I see `closePanel` as being more semantic anyways.  As this gets updated in the future with more functionality, it may be required that other things are opened/closed and in that case `closePanel` will be more descriptive? 🤔

Reply:
Then `isOpen` should be `isPanelOpen`. But, I doubt we'll need that. I think it's reasonable to assume that the panel will be the main thing that is "open" and the non-suffixed props refer to that.

Reply:
Good points.  Updated!